### PR TITLE
Added jsbeautifier to banks_description

### PR DESCRIPTION
**Pull request by eoliveros**
when creating/editing the bank_description, it uses jsbeautifier. noticeable when you enter it as one line string.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -289,3 +296,14 @@ class UtilityModelView(RestrictedModelView):
...
     )
+
+    column_formatters = {
+        'fields_description':  _formatter
+    }
+
+    def on_model_change(self, form, model, is_created):
+        opts = jsbeautifier.default_options()
+        opts.indent_size = 2
+        if is_created:
```
Comment (djpnewton) [inline models.py]: these two lines are not required

File: models.py [hunk 2]
```diff
@@ -232,6 +232,13 @@ def bank_description_check(form, field):
             raise ValidationError('"{}" is not a list/array'.format(FIELDS))
         fields_check(item[FIELDS])
 
+def _formatter(view, context, model, name):
```
Comment (djpnewton) [inline models.py]: this formatter is not required, it will get updated on the 'on_model_change' function anyhow

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: models.py (hunk 1 above)
Comment (eoliveros) [reply models.py]: the opts lines above is what set the indent size, by default its 4.

Removing anyway

File: models.py (hunk 2 above)
Comment (eoliveros) [reply models.py]: deleted the formatter

File: models.py (hunk 1 above)
Comment (djpnewton) [reply models.py]: you removed the wrong lines

```
if is_created:
   model.bank_description = jsbeautifier.beautify(form.bank_description.data)
```

these lines are not useful

Comment (eoliveros) [reply models.py]: Re-added then deleted the if lines

**Comment by djpnewton** [comment]
cool looks good, can you give me your test results?

**Comment by eoliveros** [comment]
results will be in caprover:
image

image

**Comment by djpnewton** [comment]
I mean, have you tested this functionality and is it working ok?

**Comment by eoliveros** [comment]
> I mean, have you tested this functionality and is it working ok?

screenshot is in my previous comment

**Comment by djpnewton** [comment]
the screenshot looks fine, can you tell me if you have you tested this functionality and is it working ok?

**Comment by eoliveros** [comment]
> the screenshot looks fine, can you tell me if you have you tested this functionality and is it working ok?

The result is the screenshot and it looks ok.

**Comment by djpnewton** [comment]
have you tested it? A screenshot does not tell me if you have tested the full functionality

have you edited some "untidy" JSON and it gets nicely formatted without making any functional changes to the JSON data?

**Comment by eoliveros** [comment]
> have you tested it? A screenshot does not tell me if you have tested the full functionality
> 
> have you edited some "untidy" JSON and it gets nicely formatted without making any functional changes to the JSON data?

yes, i when i entered it as one string, removing spaces in keys and values, it reformats it to the pretty version

**Pull request merged**